src/main/cliapp/src/containers/dataLoadsPage/DataLoadsComponent.js

```diff
@@ -151,7 +151,7 @@ export const DataLoadsComponent = () => {
   const loadActionBodyTemplate = (rowData) => {
     let ret = [];
 
-    ret.push(<Button key="edit" icon="pi pi-pencil" className="p-button-rounded p-button-warning mr-2" onClick={() => editLoad(rowData)} />);
+    ret.push(<Button key="edit" con="pi pi-pencil" className="p-button-rounded p-button-warning mr-2" onClick={() => editLoad(rowData)} />);
```

Comment:
@kthorat-prog something is not right with this PR as its reverting code I put in earlier this week.

Reply:
@oblodgett yes. I see that now. Need to revery commits on this file. There should be total only 6 changed files.